### ISSUE TITLE
On listing details page, retitle "Unit Features." 

### ISSUE THREAD

**Issue by iph-97** [bloom-housing/bloom#6297]
On listing details page for all listings, retitle "Unit Features."  It should be titled "Unit Features and Accessibility Type."

**Comment by sarahlazarich** [comment]
Can we do in Core as well? Pointing as if we are

**Comment by iph-97** [comment]
Sure!

**Comment by emilyjablonski** [comment]
@iph-97 Reminder to please use sentence case, i.e. `Unit features and accessibility type`! We had an unnecessary cycle on this one
